attest/core/src/ias/serial.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{VerificationReport, VerificationSignature};
+use alloc::{format, string::String, vec::Vec};
+use serde::{de::Error, Deserialize, Deserializer, Serialize, Serializer};
+
+// Intermediate representation of a VerificationReport with hex-encoded strings
+#[derive(Serialize, Deserialize)]
+struct EncodedReport {
+    pub sig: String,
+    pub chain: Vec<String>,
+    pub http_body: String,
+}
+
+/// Serializer for representing ias report signing certificate chain and
+/// report signature bytes as hex strings.
+pub fn serialize<S: Serializer>(
```

Comment:
I'd disagree because in general the Verification reports serialize in a messy format. It'd be useful for this to be accessible to other crates without having to worry about dependency collisons.